cmd/cli/install_test.go

```diff
@@ -979,9 +1091,10 @@ var _ = Describe("Test osm image pull policy cli option", func() {
 		installCmd := &installCmd{
 			osmImagePullPolicy: "Always",
 		}
-
-		vals, err := installCmd.resolveValues()
+		parsedSetOptions, parseErr := installCmd.parseSetOptions()
+		vals, err := installCmd.resolveValues(parsedSetOptions)
 		Expect(err).NotTo(HaveOccurred())
+		Expect(parseErr).NotTo(HaveOccurred())
 
```

Comment:
I haven't taken a super close look at each of these tests, but can you verify

1. There's at least one test where no `--set` options are passed
1. There's at least one test for the comma-separated case like in the help message for the flag

Reply:
#1 was already take care of..  for #2 I have added new test case